canonical/charmhub.io
# Move publisher listing view to React app

## Done
- Moved publisher listing section to React application

## How to QA
- Go to https://charmhub-io-1811.demos.haus/<CHARM_NAME>/listing
- Check that it looks and works the same as https://charmhub.io/<CHARM_NAME>/listing
- *Notes*:
  - The "A published charm is required to successfully save this form." now only shows if the charm/bundle is not published
  - The "Bundle topology" has been removed as it doesn't do anything

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-10532
